tools/db_configs/cockroachdb/create_user.sql

```diff
@@ -0,0 +1,3 @@
+CREATE USER mongooseim WITH PASSWORD 'mongooseim_secret';
+GRANT admin TO mongooseim;
+GRANT ALL ON DATABASE mongooseim to mongooseim;
```

Comment:
Unfortunately, there is no psql available on CockroachDB docker container. So it is very hard to perform this operation inside setup-db.sh. There is possibility to create database when passing the database name and username as docker container environment variables. I tried using that option, but there is a problem that cockroachdb is generating user's certificates and that operation fails if not done manually(because of short length of certificates validity). I've decided, that the best option is to create user using separate sql migration file.